### DISCUSSION TITLE
no debug information for function in dwarf

### DISCUSSION THREAD

**Discussion by luckjoke** [emscripten-core/emscripten#19019] [Q&A]
I get error 'no debug information for function' when I use "emmake make -O0 -g -gseparate-dwarf" to compile my project and run on chrome. 
    I get message "Loaded debug symbols for xxx, found 702 source file(s)" on browser and think that dwarf is loaded.
    Is there any thing I lost?

**Comment by sbc100** [comment]
Are you sure the `-O0 -g -gseparate-dwarf` are being passed the compiler and linker?  Normally one would do something more like `emmake make CFLAGS="-O0 -g -gseparate-dwarf"` but it depends on the project.
